test/recipes/15-test_out_option.t

```diff
@@ -41,6 +41,7 @@ $rand_path .= "/test.pem";
 
 test_illegal_path($rand_path);
 test_legal_path('test.pem');
+test_legal_path(File::Spec->devnull());
```

Comment:
As far as Windows go there is the "mingw" ambiguity. File::Spec->devnull() is something from Perl's "mind", which customarily has nothing to do with real Windows binary, such as one compiled by "mingw" compiler, it tries to exercise.

Reply:
... but isn't `/dev/null` something that get's converted to `nul:` by the mingw runtime when given on the command line?  (assuming that mingw perl's `File::Spec->devnull()` gives `"/dev/null"`)

Reply:
No, mingw run-time doesn't do any conversions. MSYS2 does, Cygwin does, mingw does *nothing* of the sort. Just like any other Windows executable wouldn't. Think migw build on Linux. Perl believes devnull is /dev/null, but 'wine test/whatever' lives in world of its own.

Reply:
*argh* msys2 / mingw, I find it difficult to tell them apart at times.  So ok, msys2, and ok, I get what you mean.

Reply:
Do you have a solution for this issue?  I don't...

Reply:
Omitting the test when target is mingw would be appropriate. And it would kind of consistent with check for $^O eq 'MSWin32'. You kind of say "whenever we generate Windows binaries, we don't run this test."

Reply:
So, err, wait...  at the beginning of this recipe, there is a check for `MSWin32`, which the mingw flavored perls give back for `$^O`, so at this point, I'm wondering what we're even discussing...

Reply:
Never mind, got it...  cross-compiled mingw under Linux fails, of course...  so we can skip this test when `$config{CROSS_COMPILE}` is has a non-empty string value, is that an acceptable solution?